src/DependencyInjection/Configuration.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Rokka\RokkaClientBundle\DependencyInjection;
 
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\NodeDefinition;
```

Comment:
That's not needed aynmore, right?

Reply:
yep, removed.

seems that the cs fixer is also not running anymore. we could add it to the static.yml workflow. no time right now though.